fault-quarantine/main.go

```diff
@@ -103,9 +106,18 @@ func run() error {
 	return g.Wait()
 }
 
-func parseFlags() (metricsPort, kubeconfigPath *string, dryRun, circuitBreakerEnabled *bool, tomlConfigPath *string) {
+func parseFlags() (
```

Comment:
super nit: I usually like to define flags in a var block as package level variables like this at the top of the file

```
var (
metricsPort string
databaseClientCertMountPath string
...
)
```

just so that you don't have to a have a function returning tons of things as the flags grow